src/heliclockter/__init__.py

```diff
@@ -101,21 +102,19 @@ def __init__(  # pylint: disable=unused-argument
 
             self.assert_aware_datetime(self)
 
-    if PYDANTIC_V1_AVAILABLE:
-
-        @classmethod
-        def __get_validators__(cls) -> Iterator[Callable[[Any], Optional[datetime_tz]]]:
-            yield cls._validate
+    @classmethod
+    def __get_validators__(cls) -> Iterator[Callable[[Any], Optional[datetime_tz]]]:
+        yield cls._validate
 
-        @classmethod
-        def _validate(cls: Type[DateTimeTzT], v: Any) -> Optional[DateTimeTzT]:
-            if v is None:
-                return None
+    @classmethod
+    def _validate(cls: Type[DateTimeTzT], v: Any) -> Optional[DateTimeTzT]:
+        if v is None:
+            return None
 
-            dt = v if isinstance(v, _datetime.datetime) else parse_datetime(v)
-            return cls.from_datetime(dt)
+        dt = v if isinstance(v, _datetime.datetime) else parse_datetime(v)
+        return cls.from_datetime(dt)
```

Comment:
This should be behind the `if PYDANTIC_V2_AVAILABLE` check too, no? Otherwise you can't use `parse_datetime`

Reply:
https://github.com/channable/heliclockter/pull/10/files#diff-5ecf7fd805ecf7e765e7b19bf5aa34c1f99a9ed55a3adbeb6a4db0e60075233bL22

`parse_datetime` is imported here for pydantic v1, it doesn't exist in pydantic v2 (except in `pydantic.v1`)
`_validate` is meant specifically for pydantic v1, it isn't supported in v2.

Reply:
Okay sure, but what if neither pydantic v1 nor pydantic v2 is available? then we still have a problem with this implementation, no?

I don't understand why `__get_validators__` and `_validate` no longer need to be behind some kind of pydantic available flag. We don't want to require pydantic as a dependency for heliclockter.

Reply:
> Okay sure, but what if neither pydantic v1 nor pydantic v2 is available? then we still have a problem with this implementation, no?

ahh yes that's true.

So it should be:
`if PYDANTIC_V1_AVAILABLE or PYDANTIC_V2_AVAILABLE:`

Reply:
Now it should work in all cases